src/cmd/lint.js

```diff
@@ -5,11 +5,59 @@ import {FileFilter} from './build';
 
 const log = createLogger(__filename);
 
+// Flow types
+
+export type LinterLogLevel = 'debug' | 'fatal';
+export type LinterOutputType = 'text' | 'json';
+
+export type LinterCreatorParams = {
+  config: {
+    logLevel: LinterLogLevel,
```

Comment:
for me, the less variables I have to follow the better. If I'm looking for a type definition and it points to a variable then I have to look in two places to find the definition. Obviously you need to use a variable when the definition is shared by more than one place.